releases-and-milestones/release-process.md

```diff
@@ -0,0 +1,41 @@
+<!---
+   Copyright 2022 Axis Communications AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+--->
+
+# Release process
+
+This document describes the release process for Eiffel editions.
+
+## Pre-release activities
+
+Look up the name of the next edition in the [Versioning](../eiffel-syntax-and-usage/versioning.md) document.
+
+GitHub [milestones](https://github.com/eiffel-community/eiffel/milestones?state=open) are used to track the proposed contents of an upcoming edition. Milestones are named "Edition \<name>", e.g. "Edition Arica". Issues, not pull requests, are added to the milestone.
+
+## Making a release
+
+When all issues in the scope of the edition have been closed and it's time to make the release, follow these steps:
+
+1.  Verify that all issues in the [milestone](https://github.com/eiffel-community/eiffel/milestones?state=open) are closed.
+1.  Create a pull request with the following changes (see [PR 277](https://github.com/eiffel-community/eiffel/pull/277) for reference):
+    1.  Change all "No edition set" entries in the release tables of the latest YAML file for each event type to a link to the not yet existing edition tag.
+    1.  Claim the edition in [versioning.md](../eiffel-syntax-and-usage/versioning.md), including a short summary of the changes in the edition.
+1.  When the pull request has been merged, create and push an "edition-\<name>" annotated tag (use `git tag -a`). The tag comment could include a short version of the included changes to the protocol. Any new major versions of event types should be called out.
+1.  Create a GitHub release based on the newly pushed tag. The tag comment can probably be reused as the release description.
```

Comment:
Are there any settings when creating the release e.g. https://github.com/eiffel-community/eiffel-remrem-publish/releases has different layout of the release. Any tick boxes you need to mark?

Reply:
They are very similar to me: https://github.com/eiffel-community/eiffel/releases vs https://github.com/eiffel-community/eiffel-remrem-publish/releases. What are the differences you refer to, @m-linner-ericsson ?

Reply:
![image](https://user-images.githubusercontent.com/54929896/199039680-e2744725-9eef-4b3b-a462-cad025ecfdb4.png)

The red circled parts are missing in 2.0.26

Reply:
It seems that it is possible to generate a nice looking release note once a tag is selected, and that release note will include the Contributors and maybe more interesting stuff.
I propose that we leave the current instruction as-is, and then once we have tested this feature along with the upcoming release we could possibly update the instruction to recommend/require that the release notes are generated using that button.

Reply:
Interesting, I hadn't seen that feature. Let's try it out for Arica. There are ways of [configuring what PRs to include](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#configuring-automatically-generated-release-notes) and that's something we could use to only include PRs affecting the protocol (if we start labeling issues and PRs with e.g. a "protocol" label, but I think we should do that anyway).